project/.travis/install_test.sh

```diff
@@ -16,8 +16,9 @@ chmod u+x "${HOME}/bin/phpunit"
 wget https://github.com/satooshi/php-coveralls/releases/download/v1.0.1/coveralls.phar --output-document="${HOME}/bin/coveralls"
 chmod u+x "${HOME}/bin/coveralls"
 
-# To be removed when this issue will be resolved: https://github.com/composer/composer/issues/5355
-if [ "${COMPOSER_FLAGS}" = '--prefer-lowest' ]; then
-    composer update --prefer-dist --no-interaction --prefer-stable --quiet
-fi
+# To be removed when these issues are resolved:
```

Comment:
Was trying to keep the same style as other comments on those files. I can change on all, but not sure if I should on this PR.